extensions/PhabBugz/lib/Feed.pm

```diff
@@ -393,40 +470,7 @@ sub process_revision_change {
 
   # Change is a submission of the uplift request form.
   if (_is_uplift_request_form_change($story_text)) {
-    my $revision_phid = $revision->phid;
-    INFO(
-      "Uplift request form submitted on $revision_phid, " .
-      "requesting `#release-managers` review."
-    );
-
-    # Get `#release-managers` review group.
-    my $release_managers_group = Bugzilla::Extension::PhabBugz::Project
-      ->new_from_query({name => 'release-managers'});
-
-    if (!$release_managers_group) {
-      WARN(
-        "Uplift request change detected but `#release-managers` was " .
-        "not found on Phabricator."
-      );
-      return;
-    }
-
-    my $release_managers_phid = $release_managers_group->phid;
-
-    # Request `#release-managers` review on each revision.
-    foreach my $stack_revision_phid (keys %{$revision->stack_graph_raw}) {
-      # Query Phabricator for each revision object related to the updated revision.
-      my $stack_revision = Bugzilla::Extension::PhabBugz::Revision->new_from_query(
-        {phids => [$stack_revision_phid]}
-      );
-
-      # Add `#release-managers!` review and set revision status.
-      $stack_revision->add_reviewer("blocking($release_managers_phid)");
-      $stack_revision->update();
-
-      INFO("Requested #release-managers review of $stack_revision_phid.");
-    }
-
+    process_uplift_request_form_change($revision, $changer);
     return;
```

Comment:
Just do `return process_uplift_request_form_change($revision, $changer);` here in case a return value is needed by the caller.

Reply:
What should `process_uplift_request_form_change` return?